spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before(:all) do
+    @user = User.new(name: 'John', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.',
+                     posts_counter: 0)
+  end
+
+  context 'User implementation testing' do
+    it 'User name should be present' do
+      expect(@user.name).to eq('John')
+    end
+
+    it 'Recent post method should return zero' do
+      expect(@user.three_most_recent_post.length).to be 0
+    end
+
+    it 'Post counter should be an interger' do
+      expect(@user.posts_counter).to be_a_kind_of(Numeric)
+    end
+  end
```

Comment:
- [ ] Good job on writing your tests well so far 👏 👏 However, when I tried to run the tests, I ran into an error that is shown below 👇 
![image](https://user-images.githubusercontent.com/90222110/214278657-6473896c-aff9-4067-9e9c-dd3a0457f667.png)

I tried to resolve the error by commenting the following line of code 👇 
![image](https://user-images.githubusercontent.com/90222110/214278977-328f9661-084f-499b-aa0e-24d37dbaa87e.png)
It is located in `spec/rails_helper.rb`
It worked because after commenting it and running tests, some were running and passing so please do the same so that next reviewer will be able to evaluate them 😃 👍